### DISCUSSION TITLE
Is it possible to use a custom CSS framework (other than Bootstrap) for a `website` quarto project?

### DISCUSSION THREAD

**Discussion by joelvonrotz** [quarto-dev/quarto-cli#8144] [Q&A] [answered]
### Description

Hello
Is it possible to NOT use Bootstrap for the Quarto website project?
For me Bootstrap feels a bit too bloated and I would like to use a more lightweight framework. Don't get me wrong, I like the themes and they look nice, especially `cosmo`.

When using the parameter `theme: none`, everything gets strapped such as footer, header, logo, search, etc. Is it possible to include these functions?

I'm using quarto version 1.3.450

**Comment by mcanouil** [comment, marked as answer]
Basically no as Quarto creates the website envelope not you.
So if you remove Bootstrap you also remove the components tied to it.
This being said, you can include other CSS framework into Quarto to have custom page layout, but it is not (S)CSS conflicts proof ...

**Reply by joelvonrotz** [reply]
Ah okay, thank you for your explanation!
